sample/src/androidTest/java/com/schibsted/spain/barista/sample/MenuClickActions.java

```diff
@@ -0,0 +1,53 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.BaristaAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.BaristaMenuClickActions.menuClick;
+
+@RunWith(AndroidJUnit4.class)
+public class MenuClickActions {
```

Comment:
Great test class! What do you think about naming it like a test class? I think that the other tests at Barista contain the word `Test` or `Should` in the name of the class.